tests/app/celery/test_lookup_recipient_communication_permissions_task.py

```diff
@@ -126,9 +128,69 @@ def test_recipient_has_given_permission_should_return_true_if_user_grants_permis
     )
 
     mock_task = mocker.Mock()
-    assert recipient_has_given_permission(
+    permission_message = recipient_has_given_permission(
         mock_task, 'VAPROFILEID', '1', 'some-notification-id', SMS_TYPE, 'some-communication-id'
     )
+    assert permission_message is None
+
+
+def test_recipient_has_given_permission_should_return_true_if_user_permissions_not_set_and_no_com_item(
+        client, mocker, fake_uuid
+):
+    mocked_va_profile_client = mocker.Mock(VAProfileClient)
+    mocked_va_profile_client.get_is_communication_allowed = mocker.Mock(side_effect=CommunicationItemNotFoundException)
+    mocker.patch(
+        'app.celery.lookup_recipient_communication_permissions_task.va_profile_client',
+        new=mocked_va_profile_client
+    )
+
+    mocker.patch(
+        'app.celery.lookup_recipient_communication_permissions_task.get_communication_item',
+        return_value=None
+    )
+
+    mock_task = mocker.Mock()
+    permission_message = recipient_has_given_permission(
+        mock_task, 'VAPROFILEID', '1', 'some-notification-id', SMS_TYPE, fake_uuid
+    )
+    assert permission_message is None
+
+
+@pytest.mark.parametrize(
+    ('send_indicator'),
+    [True, False],
+    ids=[
+        "default_send_indicator is True",
+        "default_send_indicator is False"
+    ]
+)
+def test_recipient_has_given_permission_with_default_send_indicator_and_no_preference_set(
+        client, mocker, send_indicator: bool
+):
+    mocked_va_profile_client = mocker.Mock(VAProfileClient)
+    mocked_va_profile_client.get_is_communication_allowed = mocker.Mock(side_effect=CommunicationItemNotFoundException)
+    mocker.patch(
+        'app.celery.lookup_recipient_communication_permissions_task.va_profile_client',
+        new=mocked_va_profile_client
+    )
+
+    mock_communication_item = mocker.Mock(CommunicationItem)
+    mock_communication_item.default_send_indicator = send_indicator
```

Comment:
This doesn't need to be mocked at all.  You could just create a new CommunicationItem instance.